### ISSUE TITLE
[feeds] charger un nouvel élément quand un ancien est lu

### ISSUE THREAD

**Issue by B4rb3rouss** [BlogoText/blogotext#357]
Quand on fait ctrl-down, ça charge un article suivant.
Cependant, quand on a finit la page (10 articles actuellement), il faut cliquer sur la flèche pour charger les articles plus anciens, ou bien recharger la page.

L'idée est de charger un nouvel élément lorsqu'un est lu pour ne pas avoir à recharger la page.

**Comment by remrem** [comment]
ça va faire beaucouuuuuup de requêtes :/
Note que tu peux augmenter le nombre d'item à afficher dans les paramètres.

**Comment by B4rb3rouss** [comment]
Pas tant que ça si l'affichage des éléments est déjà fait en AJAX.
Au lieu de charger à chaque élément lu, on peut charger automatiquement lorsqu'on est arrivé à la fin (un peu comme le fait le site walldevil.com)

**Comment by remrem** [comment]
tu veux dire comme un "infinite scroll" (genre [démo trouvée à l'arrache](https://infinite-scroll.com/demo/full-page/page5.html]) mais qui réagit au clavier ?

Je ré-ouvre le ticket si d'autres ont un avis à ce sujet, mais pour ma part, je ne suis pas fan de ce type de chargement de contenu.

**Comment by B4rb3rouss** [comment]
oui un infinite scroll c'est ça.
Si tu as d'autres idées c'est super, c'est juste qu'actuellement c'est très peu pratique de recharger un page pour lire les éléments plus anciens. Non ? Qu'en penses-tu?

**Comment by remrem** [comment]
Je n'ai pas la même utilisation que toi, du coup, ce qui peut être pertinent pour toi, ne le sera pas forcément pour moi... Je pense qu'il faut ce donner un peu de temps pour revoir l’agrégateur de flux. (j'aimerai bien revoir un peu le core du truc avant la 4.0...)

J'utilise une installation de blogotext sur un serveur local pour faire de la veille sur une centaine de site, en général j'ai entre 50 et 200 nouveaux items par jour, j'ai mis un affichage à 100 items par page, je lis ce qui m'intéresse et je marque comme lu le reste. Je garde un oeil dessus 2-3 fois par jour dessus, du coup, je n'ai que rarement une seconde page à afficher. Je vais voir pour basculer cette instance de BT sur un serveur en ligne et me forcer à utiliser un mobile pour me rapprocher d'une utilisation plus contraignante et voir ce que ça donne. Me faut un peu de temps ;)

**Comment by B4rb3rouss** [comment]
Pas de soucis :)
C'est sûr qu'en affichant 100 itemps d'un coup, il n'y a pas ce problème. On avait réduit à 10 justement parce que c'était un peu trop lent.

Je suis d'accord avec toi, plus je comprends comment est fait blogotext en parcourant le code plus je remarque des trucs qui sont bons à retirer ou à repenser pour optimiser, sans tout jeter pour autant.

**Comment by remrem** [comment]
> On avait réduit à 10 justement parce que c'était un peu trop lent.
Cette limite est configurable dans les préférences, rien de t'empêche de l'augmenter pour voir si ça te va mieux ;)

**Comment by B4rb3rouss** [comment]
yep, c'est ce que j'ai fait, et avec les flux vides cachés, c'est mieux :)

De la à fermer l'issue, je ne sais pas, ça serait quand même bien pratique un chargement automatique. Cela dit, je fais le malin mais suis incapable de le faire moi-même donc tu fermes si tu veux :)
